.atomist/editors/AddAtomistCopyrightToRug.rug

```diff
@@ -1,16 +1,34 @@
-@description "This rug is copied into your project by AddRugEditor"
-@tag "documentation"
+/*
+ * Copyright © 2016 Atomist, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@description "This rug is adds an Atomist copyright message to a file"
+@tag "rug"
+@tag "atomist"
 editor AddAtomistCopyrightToRug
 
 precondition IsRugArchive
```

Comment:
The assumption here is that the header is commented in something we know is ok for Rug, but that the comment notation might not sit well with other systems?